frontend/src/context/CartContext.test.tsx

```diff
@@ -133,6 +133,22 @@ describe("Given a user already has items in their cart", () => {
     });
   });
 
+  describe("When they add an item that already exists in their cart", () => {
```

Comment:
🐘 💭 Been having a thought that perhaps it is better and a bit clearer when reading an individual test to have `the user` in place of `they`

so that looking at the individual sentence it is clear :) 
```suggestion
  describe("When the user adds an item that already exists in their cart", () => {
```